2.6/root/usr/share/container-scripts/mongodb/post-init.sh

```diff
@@ -0,0 +1,8 @@
+# This file unset environmental variables about MongoDB users
+
+# Make sure env variables don't propagate to mongod process.
+function unset_env_vars() {
+  unset MONGODB_USER MONGODB_PASSWORD MONGODB_DATABASE MONGODB_ADMIN_PASSWORD
+}
+
+unset_env_vars
```

Comment:
I don't see the reason of having this function if it's used only once here, right after declaration.